app/ide-desktop/eslint.config.js

```diff
@@ -231,7 +231,24 @@ const RESTRICTED_SYNTAXES = [
     {
         selector:
             'JSXElement[closingElement!=null]:not(:has(.children:matches(JSXText[raw=/\\S/], :not(JSXText))))',
-        message: 'Use self-closing tags (`<tag />`) for tags without children.',
+        message: 'Use self-closing tags (`<tag />`) for tags without children',
+    },
+    {
+        // TODO [sb]: `z-3` should be eliminated, but is currently still required.
+        // TODO: this does not work for template strings and nested strings
```

Comment:
is this gonna be fixed in this PR? if not lets make it as FIXME with gh issue

Reply:
cannot be fixed until GUI1 is removed, as `z-3` *is* the workaround for GUI1 - the reason being that GUI1 uses `z-1` through `z-4` for their renderinglayers. will add gh issue.

Reply:
note: the `TODO` below is no longer accurate, so i've removed it